admin/access.php

```diff
@@ -29,7 +29,7 @@
 
 UI::show_header();
 
-switch ($_REQUEST['action']) {
```

Comment:
Aren't those changes overkill, as we already validate the incoming data hard coded by with switches?

What I can see is that there is that we slow down the code execution by adding a function that doesn't improve the security in those cases.

The filter_input function would be interesting if we had some user controlled output, like a message or a post that's saved in a database, unfiltered/unsanitized. But in these cases I don't think it's an improvement, as code is only executed if the POST or GET request matches any case. If there's no match, the default case will handle the request or if there's no default case, nothing will happen.

Reply:
Thanks for reviewing.

I haven't compared load times between the branches yet. But I do agree that I haven't checked whether all the filters are needed.

There are xss and injection abilities in develop for pretty much every page right now.

Once I've made all the code changes in core part of my process will be on testing between core <-> develop as well as speed of those functions.